__tests__/app.test.js

```diff
@@ -126,5 +124,52 @@ describe("GET /api/articles", () => {
       expect(body.msg).toBe('404: not found')
     })
   });
+});
 
-})
\ No newline at end of file
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with all comments on an article", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(( {body} ) => {
+      expect(body.comments).toHaveLength(11)
+      body.comments.forEach((comment) => {
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
Good test here. I would say that you should assert that this ID must be 1 so that you are checking that you aren't getting comments from other articles. 🙂